packages/awslint/lib/rules/construct.ts

```diff
@@ -276,3 +285,10 @@ constructLinter.add({
     }
   },
 });
+
+function baseConstructAddendum(): string {
+  if (!process.env.AWSLINT_BASE_CONSTRUCT) {
+    return 'If the construct is using the "constructs" module, set the environment variable "AWSLINT_BASE_CONSTRUCT" and re-run';
```

Comment:
"If the module"

Reply:
This message is going to be printed along side the violating construct class, hence "if the construct"